apps.yaml

```diff
@@ -220,3 +220,18 @@ HierCVAE:
         - inverse-design
         - ml-metadata
         - wp11
+tomato:
+    metadata:
+        title: yadg
+        description: |
+            yadg: yet another datagram. FAIR data parser for time-resolved experimental data. Part of the dgbowl suite of tools for digital (electro-)catalysis and battery materials research.
+        authors: Peter Kraus, Nicolas Vetsch
+        documentation_url: https://dgbowl.github.com/yadg
```

Comment:
I got a 404 when trying to access the documentation. Please point to the README file and update the link later. We want to avoid having dead links in the registry.

Reply:
Whoops, it would help if I remembered where my own docs are.